package.json

```diff
@@ -13,7 +13,8 @@
     "bech32": "^2.0.0",
     "dotenv": "^10.0.0",
     "fastify": "^3.20.2",
-    "una-wrapper": "^0.0.7-beta"
+    "una-wrapper": "^0.0.10-beta",
```

Comment:
I deployed a new version of Una with your changes, can you change it to `^0.0.11-beta`?